docs/imcce/imcce.rst

```diff
@@ -38,23 +38,15 @@ looks like this:
    >>> field = SkyCoord(0*u.deg, 0*u.deg)
    >>> epoch = Time('2019-05-29 21:42', format='iso')
    >>> Skybot.cone_search(field, 5*u.arcmin, epoch)
-    <QTable length=12>
```

Comment:
This example seems to be very dependent on the versions (e.g. np 1.23 vs 1.24), so a bit more robust solution will be needed here.

Reply:
Would this be worth an ``IGNORE_INPUT`` then?
Like with HEASARC I just got too used to having broken up tests and just spotted 1 issue in the action logs so just handled the one.
I have fixes I can commit for imccre using np 1.24, but I'll need to mess with 1.23 if necessary.

Reply:
Tested the cone search with np 1.23 and 1.24 versions, so I'll commit and keep working through the failures with imcce.

Reply:
The output can just be restricted with say...

    >>> results  = Skybot.cone_search(field, 5*u.arcmin, epoch)
    >>> results[:4]pprint(max_width=100)

I only use 4 here out of convenience. 
Similar row output restrictions will need to be made with the other examples for imcce.
For example with Miride the example could be changed so that it will query ephemerides for asteroid
Pallas over an entire month, or maybe week (rather than year) with a time step of 1 day. 
And the output can be restricted to the corresponding scale.

Although the version sensitivity seems to be a broader issue with the docs.
For example from the docs for eso:

```
eso.ROW_LIMIT = -1   # Return all results
table = eso.query_main(column_filters={'instrument': 'APICAM', 'filter_path': 'LUMINANCE',
                                       'stime':'2019-04-26', 'etime':'2019-04-27'}, cache=False)
print(len(table))
207
print(table.columns)
<TableColumns names=('OBJECT','RA','DEC','Program_ID','Instrument','Category','Type','Mode','Dataset ID','Release_Date','TPL ID','TPL START','Exptime','Exposure','filter_lambda_min','filter_lambda_max','MJD-OBS','Airmass','DIMM Seeing at Start')>
table.pprint(max_width=100)
 OBJECT      RA         DEC      Program_ID  ...   MJD-OBS    Airmass DIMM Seeing at Start
------- ----------- ----------- ------------ ... ------------ ------- --------------------
ALL SKY 09:18:37.39 -24:32:32.7 60.A-9008(A) ... 58599.987766     1.0                  N/A
ALL SKY 09:21:07.68 -24:32:30.1 60.A-9008(A) ... 58599.989502     1.0                  N/A
ALL SKY 09:23:38.98 -24:32:27.5 60.A-9008(A) ...  58599.99125     1.0                  N/A
ALL SKY 09:26:10.28 -24:32:24.9 60.A-9008(A) ... 58599.992998     1.0                  N/A
ALL SKY 09:28:40.58 -24:32:22.4 60.A-9008(A) ... 58599.994734     1.0                  N/A
ALL SKY 09:31:43.93 -24:32:19.4 60.A-9008(A) ... 58599.996852     1.0                  N/A
ALL SKY 09:34:15.23 -24:32:17.0 60.A-9008(A) ...   58599.9986     1.0                  N/A
ALL SKY 09:36:47.53 -24:32:14.5 60.A-9008(A) ... 58600.000359     1.0                  N/A
ALL SKY 09:39:18.82 -24:32:12.2 60.A-9008(A) ... 58600.002106     1.0                  N/A
ALL SKY 09:41:49.11 -24:32:09.9 60.A-9008(A) ... 58600.003843     1.0                  N/A
    ...         ...         ...          ... ...          ...     ...                  ...
ALL SKY 19:07:39.21 -24:39:35.1 60.A-9008(A) ... 58600.395914     1.0                  N/A
ALL SKY 19:10:11.68 -24:39:39.1 60.A-9008(A) ... 58600.397674     1.0                  N/A
ALL SKY 19:12:44.15 -24:39:43.2 60.A-9008(A) ... 58600.399433     1.0                  N/A
ALL SKY 19:15:15.62 -24:39:47.1 60.A-9008(A) ... 58600.401181     1.0                  N/A
ALL SKY 19:17:46.09 -24:39:51.1 60.A-9008(A) ... 58600.402917     1.0                  N/A
ALL SKY 19:20:46.65 -24:39:55.8 60.A-9008(A) ...    58600.405     1.0                  N/A
ALL SKY 19:23:18.12 -24:39:59.7 60.A-9008(A) ... 58600.406748     1.0                  N/A
ALL SKY 19:25:51.60 -24:40:03.7 60.A-9008(A) ... 58600.408519     1.0                  N/A
ALL SKY 19:28:22.08 -24:40:07.6 60.A-9008(A) ... 58600.410255     1.0                  N/A
ALL SKY 19:30:52.55 -24:40:11.4 60.A-9008(A) ... 58600.411991     1.0                  N/A
Length = 207 rows
```

Reply:
Cut the outputs with ellipsis rather than in the command itself. We tend to keep 3-5 first and last lines.

As for version-dependent stuff, I would just avoid changing the outputs for now, and would rather try to find the underlying reason for the changes.